crypto/fipsmodule/rand/ctrdrbg.c

```diff
@@ -149,6 +385,43 @@ int CTR_DRBG_reseed(CTR_DRBG_STATE *drbg,
   return 1;
 }
 
+int CTR_DRBG_reseed_df(CTR_DRBG_STATE *drbg,
+                    const uint8_t entropy[CTR_DRBG_ENTROPY_LEN],
+                    const uint8_t *additional_data,
+                    size_t additional_data_len) {
```

Comment:
is this stuff from 8.7.1 or 8.7.2? if neither where is defined in the spec?

Reply:
I see, the (maximum allowed) length is governed by `Table 3: Definitions for the CTR_DRBG` in SP800-90A. The upper bound is 2^32 bytes.